api/swagger.yml

```diff
@@ -1250,11 +1250,14 @@ components:
           description: Path type, can either be 'common_prefix' or 'object'
         path:
           type: string
-          description: A source location to import path or to a single object. Must match the lakeFS installation blockstore type.
+          description: A source location to a prefix or to a single object. Must match the lakeFS installation blockstore type.
```

Comment:
I think we should specify 'common prefix' as we do not support prefix as defined by AWS

Reply:
Done.